math/src/main/java/com/powsybl/math/graph/UndirectedGraph.java

```diff
@@ -36,20 +36,31 @@ public interface UndirectedGraph<V, E> {
      */
     int addVertex();
 
+    /**
+     * Create a new vertex and could notify the {@link UndirectedGraphListener}s.
```

Comment:
I'd simply remove the could part of the javadoc, "could" sounds like randomly doing it. It's clearer and enough in the parameter description.

Reply:
Done